megamek/src/megamek/common/Terrain.java

```diff
@@ -399,6 +399,14 @@ public int movementCost(Entity e) {
                                 && (moveMode == EntityMovementMode.INF_LEG))) {
                     mp -= 1;
                 }
+                if (e.getCrew().getOptions().booleanOption(OptionsConstants.PILOT_ANIMAL_MIMIC)) {
+                    if (e.entityIsQuad()) {
+                        mp -= 1;
+                    }
+                    if ((moveMode == EntityMovementMode.BIPED) && e.hasQuirk("animalistic")) {
+                        mp -= 1;
+                    }
```

Comment:
You should be able to combine these if statements.

Reply:
so
if quad or biped with animalistic mp -= 1?
I wasn't sure about doing it that way as I initially wanted to keep the quads and bipeds separate.

Reply:
`if (e.entityIsQuad() || ((EntityMovementMode.BIPED == moveMode) && e.hasQuirk(OptionsConstants.QUIRK_POS_ANIMALISTIC)) {`